src/main/java/com/minecolonies/coremod/client/gui/WindowTownHall.java

```diff
@@ -931,6 +939,60 @@ private void createAndSetStatistics()
         findPaneOfTypeByID(COOKS_LABEL, Label.class).setLabelText(numberOfcooks);
     }
 
+    /**
+     * Creates several statistics and sets them in the townHall GUI.
+     */
+    private void createAndSetStatistics()
+    {
+        final int citizensSize = townHall.getColony().getCitizens().size();
+
+        Map<String, Integer> jobCountMap = new HashMap<String, Integer>();
+        for (@NotNull final CitizenDataView citizen : citizens)
+        {
+            int length = citizen.getJob().split("\\.").length;
+            String job = citizen.getJob().split("\\.")[length  - 1].toLowerCase();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define the locale to be used in this String operation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1449)